stdlib/sql/to.go

```diff
@@ -268,11 +268,17 @@ func getTranslationFunc(driverName string) (func() translationFunc, error) {
 		return MysqlColumnTranslateFunc, nil
 	case "snowflake":
 		return SnowflakeColumnTranslateFunc, nil
+	case "awsathena": // read-only support for AWS Athena (see awsathena.go)
+		return nil, errors.Newf(codes.Invalid, "writing is not supported for %s", driverName)
 	default:
 		return nil, errors.Newf(codes.Internal, "invalid driverName: %s", driverName)
 	}
 }
 
+func supportsTx(driverName string) bool {
```

Comment:
This is a good refactor for this case. Thanks!

Reply:
My pleasure.